tests/security_features_test.go

```diff
@@ -81,18 +84,48 @@ var _ = Describe("SecurityFeatures", func() {
 				}
 				Expect(*container.SecurityContext.Privileged).To(BeFalse())
 			})
-		})
 
-		Context("With selinuxLauncherType defined", func() {
-			var superPrivilegedType string
+			It("[test_id:4297]Make sure qemu processes are MCS constrained", func() {
 
-			BeforeEach(func() {
-				superPrivilegedType = "spc_t"
-				tests.UpdateClusterConfigValueAndWait(virtconfig.SELinuxLauncherTypeKey, superPrivilegedType)
-				vmi = tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+				By("Starting a VirtualMachineInstance")
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				tests.WaitForSuccessfulVMIStart(vmi)
+
+				pod := tests.GetRunningPodByVirtualMachineInstance(vmi, tests.NamespaceTestDefault)
+
+				qemuProcessSelinuxContext, err := tests.ExecuteCommandOnPod(
+					virtClient,
+					pod,
+					"compute",
+					[]string{"/usr/bin/bash", "-c", "ps -efZ | grep [/]usr/libexec/qemu-kvm | awk '{print $1}'"},
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Checking that qemu-kvm process is of the SELinux type container_t")
+				Expect(strings.Split(qemuProcessSelinuxContext, ":")[2]).To(Equal("container_t"))
+
+				By("Checking that qemu-kvm process has SELinux category_set")
+				Expect(len(strings.Split(qemuProcessSelinuxContext, ":"))).To(Equal(5))
+
+				err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Delete(vmi.Name, &metav1.DeleteOptions{})
+				Expect(err).ToNot(HaveOccurred())
 			})
+		})
+
+		Context("With selinuxLauncherType defined as spc_t", func() {
```

Comment:
Nit: spc_t tests happen to pass on Kind because everything is spc_t there... But it's really not testing anything so I would recommend skipping this as well on Kind.

Reply:
ok, let me skip that as well.